template/index.js

```diff
@@ -0,0 +1,42 @@
+import { File } from "@asyncapi/generator-react-sdk";
+import filter from  '../filters/all.js';
+import { HTML, Head, Body } from "../components/common";
+
+export default function Index({ asyncapi, params }) {
+  return (
+    <File name="index.html">
+      <HTML>
+        <Head
+          title={`${asyncapi.info().title()} ${asyncapi.info().version()} documentation`}
+          cssLinks={params.singleFile ? [filter.includeFile("template/css/global.min.css"),filter.includeFile("template/css/asyncapi.min.css")] : ["css/global.min.css","css/asyncapi.min.css"]}
+          base={params.baseHref || ""}
+        />
+        <Body>
+          <div id="root">{filter.renderSpec(asyncapi,params)}</div>
+          <Scripts params={params} />
+        </Body>
+      </HTML>
+    </File>
+  );
+}
+function Scripts({ params }) {
+  if (params.singleFile) {
+    return `
+        <script src=${filter.includeFile("template/js/asyncapi-ui.min.js")} type="text/javascript" ></script>
```

Comment:
Why do you wanna include file inside `src`? You include content of that file, not path to that file. Please fix it.